framework/wazuh/core/wdb.py

```diff
@@ -189,7 +189,7 @@ def execute(self, query, count=False, delete=False, update=False):
         """
         def send_request_to_wdb(query_lower, step, off, response):
             try:
-                request = "{} limit {} offset {}".format(query_lower, step, off)
+                request = query_lower.replace(':limit', 'limit ' + str(step)).replace(':offset', 'offset ' + str(off))
```

Comment:
Do not use + to join strings

Reply:
Changed to .format in order to improve efficiency